admin-center/display/display-overview.md

```diff
@@ -15,9 +15,10 @@ Find it in the Admin Center: [Search.gov Home]({{ site.baseurl }}/index.html) >
 Related resource: [Comparison of our hosted search results page and our results API]({{ site.baseurl }}/admin-center/display/hosted-vs-api-results.html)
 
 ## Facets
-
 > These features are only available on the hosted search results page.
 
+{% include featured-accordion.html content="facet-renaming" accordion=false %}
```

Comment:
Can we move this one up to just below the heading, like on the other pages?

Reply:
Thanks - I agree it looks a little odd. what do you think about leaving it under the header but removing the indentation?

Reply:
Removing the indentation for this type of block across the page? Yeah, that would be easy.

Reply:
looks great, thanks